samples/bluetooth/mesh/light/Kconfig

```diff
@@ -0,0 +1,21 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+source "Kconfig.zephyr"
+
+menu "Bluetooth mesh light sample"
+
+config BT_MESH_LIGHT_SAMPLE_NRF52_DFU
```

Comment:
Do we really need this config option. Can we solve it some other way by using some already existing configuration option that is in the overlay-dfu.conf file.

Reply:
Well, if we want all platforms to run this, except thingy53, we would need to exclude `CONFIG_BOARD_THINGY53_NRF5340_CPUAPP
` and `CONFIG_BOARD_THINGY53_NRF5340_CPUAPP_NS` when running the init of smp_dfu. That would work, but either way we turn this it will look a bit ugly if we are not altering the board files for thingy53